.github/dependabot.yml

```
@@ -0,0 +1,45 @@
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  # /Gemfile.lock
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "thursday"
+      time: "09:00"
+    open-pull-requests-limit: 5
+    groups:
+      rails:
+        patterns:
+          - "rails*"
+          - "actionpack"
```

why not add all the rails packages, if there is a new rails release, we'll get x Prs?
```suggestion
          - "actionpack"
          - "actioncable"
          - "actionmailbox"
          - "actionmailer"
          - "actiontext"
          - "actionview"
          - "activestorage"
          - "railties"
          - "activemodel"
          - "activejob"
```

Also, we would need to add the group `rspec`